tests/unit/algos/list/tools.js

```diff
@@ -23,3 +26,79 @@ describe('checkLimit function', () => {
         });
     });
 });
+
+describe('listingParamsMasterKeysV0ToV1', () => {
+    const testCases = [
+        {
+            v0params: {},
+            v1params: {
+                gte: DbPrefixes.Master,
+                lt: inc(DbPrefixes.Master),
+            },
+        },
+        {
+            v0params: {
+                gt: 'foo/bar',
+            },
+            v1params: {
+                gt: `${DbPrefixes.Master}foo/bar`,
```

Comment:
kinda feels to be like there should be a slash before foo for `${Master}/foo/bar` but that might just be me.

Reply:
No slash, `DbPrefixes.Master` is only a db key prefix, it does not need a delimiter.